flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -106,6 +106,108 @@ static bool isAbsent(const fir::ExtendedValue &exv) {
   return !fir::getBase(exv);
 }
 
+/// Process calls to Minloc or Maxloc intrinsic functions
+template <typename FN, typename FD>
+static fir::ExtendedValue
+genMinMaxloc(FN func, FD funcDim, mlir::Type resultType,
```

Comment:
nit: In f18 you'll see "extremum" used to mean either min or max. It seems like that might fit well here.